profiler/profiler.go

```diff
@@ -151,9 +151,8 @@ type Config struct {
 	// When true, collecting the goroutine profiles is disabled.
 	NoGoroutineProfiling bool
 
-	// When true, the agent sends all telemetries via OpenCensus exporter
-	// via OpenCensus exporter, which can be viewed in Cloud Trace and Cloud Monitoring.
-	// Default is false.
+	// When true, the agent sends all telemetries via OpenCensus exporter, which
+	// can be viewed in Cloud Trace and Cloud Monitoring. Default is false.
```

Comment:
Better to have default value on a separate line.

Reply:
ACK. Fixed.